sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Azure.AI.AnomalyDetector.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Cognitive Services Anomaly Detector client library</Description>
     <AssemblyTitle>Microsoft Azure.AI.AnomalyDetector client library</AssemblyTitle>
-    <Version>3.0.0-preview.2</Version>
+    <Version>3.0.0-beta.3</Version>
```

Comment:
We already shipped this as preview so we cannot change it to beta as it will be lower version. We should stick with preview until we GA this library.